### ISSUE TITLE
a style for record constructors

### ISSUE THREAD

**Issue by mechvel** [agda/agda-stdlib#983]
I suggest the following style for the constructors in standard records: 
to append the upper index ``ᶜ`` to the record name (with lower case for the first letter).
For example:
to set  ``divModᶜ``  for the record DivMod (instead of ``result``),
``gcdᶜ``  for the GCD (instead of ``is``),
``semigroupᶜ`` for the record Semigroup -- if we decide to use the record constructor here,
and so on.
The mnemonic of ``ᶜ`` is "constructor". 
This will automatically un-clash constructor names and make it easy to recall a such a name.

**Comment by laMudri** [comment]
In my projects, I tend to go for a `mk` prefix (`mkDivMod`, `mkGCD`, &c), short for “make”. I believe this (or rather a `Mk` prefix) has become standard in Idris, and it also already exists in some places in agda-stdlib (`mkZipper`, `mkℚ`).

**Comment by Zambonifofex** [comment]
I don’t think it makes sense to have *all* constructors follow this kind of naming convention at all, though!

Personally, I think some record constructors benefit a lot from having suggestive names to what they mean — not only to the types they construct.

**Comment by mechvel** [comment]
Yes.

**Comment by jamesmckinna** [comment]
Six years on, I think it's time to close this?
